Gruntfile.js

```diff
@@ -278,6 +278,10 @@ module.exports = function(grunt) {
       translations: {
         files: ['translations/*'],
         tasks: ['appcache', 'deploy']
+      },
+      node_modules: {
+        files: ['node_modules/**/*'],
```

Comment:
So not disagreeing with the merge, but just for another data point I often find myself killing `grunt dev` when I'm not using it, because it idles using ~15% CPU on my machine, presumably because however it's doing the watching isn't amazingly efficient. I'll let you know if it gets horribly worse.